lib/core/Observer.ts

```diff
@@ -1,12 +1,15 @@
 import IBlockchain from './interfaces/IBlockchain';
+import IOperationRateLimiter from './interfaces/IOperationRateLimiter';
 import IOperationStore from './interfaces/IOperationStore';
 import ITransactionProcessor from './interfaces/ITransactionProcessor';
 import ITransactionStore from './interfaces/ITransactionStore';
 import IUnresolvableTransactionStore from './interfaces/IUnresolvableTransactionStore';
 import IVersionManager from './interfaces/IVersionManager';
+import OperationRateLimiter from './versions/latest/OperationRateLimiter';
```

Comment:
Design: I have yet to read through the code. But it seems undesirable that this layer references any versioned code.

Reply:
O operation rate limiter shouldn't be versioned anymore. I will move it out.

Reply:
As discussed offline, we'll need to version rate limiting algorithm.